### ISSUE TITLE
Extension fails if used file provider other than file

### ISSUE THREAD

**Issue by SqrTT** [shinnn/vscode-stylelint#266]
```
(node:11005) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at stripTrailingSep (/home/sqrtt/.vscode/extensions/shinnn.stylelint-0.48.0/node_modules/path-is-inside/lib/path-is-inside.js:25:25)
    at module.exports (/home/sqrtt/.vscode/extensions/shinnn.stylelint-0.48.0/node_modules/path-is-inside/lib/path-is-inside.js:9:23)
    at validate (/home/sqrtt/.vscode/extensions/shinnn.stylelint-0.48.0/server.js:36:9)
```
Extension fails because ` Files.uriToFilePath(uri);` return `undefined` for non local files (i.e. ftp/http etx), (https://github.com/shinnn/vscode-stylelint/blob/163ea3a6bff5c1a132177d9945d27e444037280d/server.js#L34)

**Comment by shinnn** [comment]
VS Code doesn't support opening remote files yet. https://github.com/Microsoft/vscode/issues/29194

**Comment by SqrTT** [comment]
Probably I express a bit incorrectly. VSCode supports other `schema's` than `file` (via `registerFileSystemProvider`) so it's possible to address/access to file on ftp. And if such provider is registered vscode-stylelint can't start due to mentioned error.

PS. Microsoft/vscode#29194 issue is about different thing. It's about connecting to remote instance of VScode (something similar to Live Share)

**Comment by SqrTT** [comment]
fix is pretty fix is pretty straightforward #267

**Comment by shinnn** [comment]
> PS. Microsoft/vscode#29194 issue is about different thing. It's about connecting to remote instance of VScode (something similar to Live Share)

Thanks for the explanation.
